### ISSUE TITLE
The Great Pyramid - Bugs list and missing dialogue line

### ISSUE THREAD

**Issue by Leo-89** [XProger/OpenLara#306]
Also this level seems very well integrated in Open Lara engine, with the exception of:

1-One secret which is impossible to reach without "O" cause the wall in which Lara is supposed to grab in the original version here doesn't work as it should cause Lara can reach it but she can't move to the right.
![InkedOpenLara 20_01_2021 12_39_01_LI](https://user-images.githubusercontent.com/62290297/105177171-987ac680-5b26-11eb-87c8-ead3d6c69574.jpg)

2-Swan dive/jump distance in OpenLara
I don't remember if in the original game the distance you needed to jump exactly in the water pool from the high in this point was the same, but for sure in this level you need two blocks of space in order to fall exactly in the water executing Lara's swan dive from the high.
![InkedInkedOpenLara 20_01_2021 12_58_09_LI](https://user-images.githubusercontent.com/62290297/105177815-8188a400-5b27-11eb-878d-1f4183c65968.jpg)
![InkedOpenLara 20_01_2021 12_59_59_LI](https://user-images.githubusercontent.com/62290297/105177821-83526780-5b27-11eb-8e4c-d9e478f8477f.jpg)

3-Natla's dialogue line subtitles missing
As for the skater boy in Natla's mine here there is the same problem with the final boss, Natla, when she revives (dialogue line 287/54). Her dialogue line doesn't display in subtitles. There should be something wrong. :/// 
![OpenLara 20_01_2021 13_08_48](https://user-images.githubusercontent.com/62290297/105178339-2b683080-5b28-11eb-9d9b-a5cb86b54410.png)

**Comment by soaresden** [comment]
Agreed with all of these on my 3DS, for the first one, actually I managed to get it right : 
![image](https://user-images.githubusercontent.com/54243866/190353962-27502ef1-6c2c-4065-9cf0-3b0ca461a9da.png)

You have to grab, then  release and regrab and you should be OK to move
